<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
<title>. CellMix 1.5.1</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="author" content="Renaud Gaujoux, Cathal Seoighe">

<!-- bootstrap css -->
<link href="css/bootstrap.css" rel="stylesheet">
<link href="css/bootstrap-responsive.css" rel="stylesheet">
<!-- highlighting css -->
<link href="css/highlight.css" rel="stylesheet">
<!-- custom css -->
<link href="css/staticdocs.css" rel="stylesheet">

<!-- bootstrap javascript -->
<script src="js/jquery.min.js"></script>
<script src="js/bootstrap.min.js"></script>
<!-- d3 word cloud -->
<script src="js/d3.v2.min.js"></script>
<script src="js/d3.layout.cloud.js"></script>

<!--[if lt IE 9]>
  <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
<![endif]-->
  </head>

  <body>
    <div class="navbar">
  <div class="navbar-inner">
    <div class = "container">
      <a class="brand" href="index.html">CellMix 1.5.1</a>
      <div class="nav">
        <ul class="nav">
        <li><a href=""><i class="icon-home icon-white"></i> Index</a></li>
       <li><a href="_MAN.html">Documentation</a></li>
       <li><a href="_DEMOS.html">Demos</a></li>
       <li><a href="_REFERENCES.html">References</a></li>
       </ul>
   	  </div>
    </div>
  </div>
</div>

    <div class="container">
      <header>
        
      </header>
      
      <h1>References</h1><br />

<div class="row"><div class="span8">
  <ol>
   		<li><p>Abbas AR, Baldwin D, Ma Y, Ouyang W, Gurney A, Martin F, Fong S, van Lookeren Campagne M, Godowski P, Williams PM, Chan aC and Clark HF (2005).
&ldquo;Immune response in silico (IRIS): immune-specific genes identified from a compendium of microarray expression data.&rdquo;
<EM>Genes and immunity</EM>, <B>6</B>(4), pp. 319&ndash;31.
ISSN 1466-4879, <a href="http://dx.doi.org/10.1038/sj.gene.6364173">http://dx.doi.org/10.1038/sj.gene.6364173</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/15789058">http://www.ncbi.nlm.nih.gov/pubmed/15789058</a>.</li>
   		<li><p>Abbas AR, Wolslegel K, Seshasayee D, Modrusan Z and Clark HF (2009).
&ldquo;Deconvolution of blood microarray data identifies cellular activation patterns in systemic lupus erythematosus.&rdquo;
<EM>PloS one</EM>, <B>4</B>(7), pp. e6098.
ISSN 1932-6203, <a href="http://dx.doi.org/10.1371/journal.pone.0006098">http://dx.doi.org/10.1371/journal.pone.0006098</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/19568420">http://www.ncbi.nlm.nih.gov/pubmed/19568420</a>.</li>
   		<li><p>Alibés A, Yankilevich P, Cañada A and D\'\iaz-Uriarte R (2007).
&ldquo;IDconverter and IDClight: conversion and annotation of gene and protein IDs.&rdquo;
<EM>BMC bioinformatics</EM>, <B>8</B>, pp. 9.
ISSN 1471-2105, <a href="http://dx.doi.org/10.1186/1471-2105-8-9">http://dx.doi.org/10.1186/1471-2105-8-9</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1779800\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1779800\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Allen JD, Wang S, Chen M, Girard L, Minna JD, Xie Y and Xiao G (2011).
&ldquo;Probe mapping across multiple microarray platforms.&rdquo;
<EM>Briefings in bioinformatics</EM>.
ISSN 1477-4054, <a href="http://dx.doi.org/10.1093/bib/bbr076">http://dx.doi.org/10.1093/bib/bbr076</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/22199380">http://www.ncbi.nlm.nih.gov/pubmed/22199380</a>.</li>
   		<li><p>Alon U, Barkai N, Notterman Da, Gish K, Ybarra S, Mack D and Levine aJ (1999).
&ldquo;Broad patterns of gene expression revealed by clustering analysis of tumor and normal colon tissues probed by oligonucleotide arrays.&rdquo;
<EM>Proceedings of the National Academy of Sciences of the United States of America</EM>, <B>96</B>(12), pp. 6745&ndash;50.
ISSN 0027-8424, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=21986&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=21986&amp;tool=pmcentrez&amp;rendertype=abstract</a>.</li>
   		<li><p>Barrett T, Troup DB, Wilhite SE, Ledoux P, Evangelista C, Kim IF, Tomashevsky M, Marshall Ka, Phillippy KH, Sherman PM, Muertter RN, Holko M, Ayanbule O, Yefanov A and Soboleva A (2010).
&ldquo;NCBI GEO: archive for functional genomics data sets&ndash;10 years on.&rdquo;
<EM>Nucleic acids research</EM>, <B>39</B>(November 2010), pp. 1005&ndash;1010.
ISSN 1362-4962, <a href="http://dx.doi.org/10.1093/nar/gkq1184">http://dx.doi.org/10.1093/nar/gkq1184</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/21097893">http://www.ncbi.nlm.nih.gov/pubmed/21097893</a>.</li>
   		<li><p>Bertsekas D (1999).
<EM>Nonlinear programming</EM>.
Athena Scientific.
ISBN 1-886529-00-0, <a href="http://www.athenasc.com/nonlinbook.html">http://www.athenasc.com/nonlinbook.html</a>.</li>
   		<li><p>Boguski MS, Lowe TM and Tolstoshev CM (1993).
&ldquo;dbEST&ndash;database for &quot;expressed sequence tags&quot;.&rdquo;
<EM>Nature genetics</EM>, <B>4</B>(4), pp. 332&ndash;3.
ISSN 1061-4036, <a href="http://dx.doi.org/10.1038/ng0893-332">http://dx.doi.org/10.1038/ng0893-332</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/8401577">http://www.ncbi.nlm.nih.gov/pubmed/8401577</a>.</li>
   		<li><p>Bolen CR, Uduman M and Kleinstein SH (2011).
&ldquo;Cell Subset Prediction for Blood Genomic Studies.&rdquo;
<EM>BMC Bioinformatics</EM>, <B>12</B>(1), pp. 258.
ISSN 1471-2105, <a href="http://dx.doi.org/10.1186/1471-2105-12-258">http://dx.doi.org/10.1186/1471-2105-12-258</a>, <a href="http://www.biomedcentral.com/1471-2105/12/258">http://www.biomedcentral.com/1471-2105/12/258</a>.</li>
   		<li><p>Bolstad BM, Irizarry Ra, Astrand M and Speed TP (2003).
&ldquo;A comparison of normalization methods for high density oligonucleotide array data based on variance and bias.&rdquo;
<EM>Bioinformatics (Oxford, England)</EM>, <B>19</B>(2), pp. 185&ndash;93.
ISSN 1367-4803, <a href="http://www.ncbi.nlm.nih.gov/pubmed/12538238">http://www.ncbi.nlm.nih.gov/pubmed/12538238</a>.</li>
   		<li><p>Brazma A, Hingamp P, Quackenbush J, Sherlock G, Spellman P, Stoeckert C, Aach J, Ansorge W, Ball CA, Causton HC, Gaasterland T, Glenisson P, Holstege FC, Kim IF, Markowitz V, Matese JC, Parkinson H, Robinson A, Sarkans U, Schulze-Kremer S, Stewart J, Taylor R, Vilo J and Vingron M (2001).
&ldquo;Minimum information about a microarray experiment (MIAME)-toward standards for microarray data.&rdquo;
<EM>Nature genetics</EM>, <B>29</B>(4), pp. 365&ndash;71.
ISSN 1061-4036, <a href="http://dx.doi.org/10.1038/ng1201-365">http://dx.doi.org/10.1038/ng1201-365</a>, <a href="http://dx.doi.org/10.1038/ng1201-365">http://dx.doi.org/10.1038/ng1201-365</a>.</li>
   		<li><p>Breiman L (2001).
&ldquo;Random forests.&rdquo;
<EM>Machine learning</EM>, <B>45</B>, pp. 5&ndash;32.
<a href="http://www.springerlink.com/index/U0P06167N6173512.pdf">http://www.springerlink.com/index/U0P06167N6173512.pdf</a>.</li>
   		<li><p>Brunet J, Tamayo P, Golub TR and Mesirov JP (2004).
&ldquo;Metagenes and molecular pattern discovery using matrix factorization.&rdquo;
<EM>Proceedings of the National Academy of Sciences of the United States of America</EM>, <B>101</B>(12), pp. 4164&ndash;9.
ISSN 0027-8424, <a href="http://dx.doi.org/10.1073/pnas.0308531101">http://dx.doi.org/10.1073/pnas.0308531101</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/15016911">http://www.ncbi.nlm.nih.gov/pubmed/15016911</a>.</li>
   		<li><p>Bussey KJ, Kane D, Sunshine M, Narasimhan S, Nishizuka S, Reinhold WC, Zeeberg B, Ajay W and Weinstein JN (2003).
&ldquo;MatchMiner: a tool for batch navigation among gene and gene product identifiers.&rdquo;
<EM>Genome biology</EM>, <B>4</B>(4), pp. R27.
ISSN 1465-6914, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=154578\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=154578\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Carter SL, Eklund AC, Mecham BH, Kohane IS and Szallasi Z (2005).
&ldquo;Redefinition of Affymetrix probe sets by sequence overlap with cDNA microarray probes reduces cross-platform inconsistencies in cancer-associated gene expression measurements.&rdquo;
<EM>BMC bioinformatics</EM>, <B>6</B>, pp. 107.
ISSN 1471-2105, <a href="http://dx.doi.org/10.1186/1471-2105-6-107">http://dx.doi.org/10.1186/1471-2105-6-107</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1127107\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1127107\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Clarke J, Seo P and Clarke B (2010).
&ldquo;Statistical expression deconvolution from mixed tissue samples.&rdquo;
<EM>Bioinformatics (Oxford, England)</EM>, <B>26</B>(8), pp. 1043&ndash;9.
ISSN 1367-4811, <a href="http://dx.doi.org/10.1093/bioinformatics/btq097">http://dx.doi.org/10.1093/bioinformatics/btq097</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/20202973">http://www.ncbi.nlm.nih.gov/pubmed/20202973</a>.</li>
   		<li><p>Cobb JP, Mindrinos MN, Miller-Graziano C, Calvano SE, Baker HV, Xiao W, Laudanski K, Brownstein BH, Elson CM, Hayden DL, Herndon DN, Lowry SF, Maier RV, Schoenfeld Da, Moldawer LL, Davis RW, Tompkins RG, Bankey P, Billiar T, Camp D, Chaudry I, Freeman B, Gamelli R, Gibran N, Harbrecht B, Heagy W, Heimbach D, Horton J, Hunt J, Lederer J, Mannick J, McKinley B, Minei J, Moore E, Moore F, Munford R, Nathens A, O'keefe G, Purdue G, Rahme L, Remick D, Sailors M, Shapiro M, Silver G, Smith R, Stephanopoulos G, Stormo G, Toner M, Warren S, West M, Wolfe S and Young V (2005).
&ldquo;Application of genome-wide expression analysis to human health and disease.&rdquo;
<EM>Proceedings of the National Academy of Sciences of the United States of America</EM>, <B>102</B>(13), pp. 4801&ndash;6.
ISSN 0027-8424, <a href="http://dx.doi.org/10.1073/pnas.0409768102">http://dx.doi.org/10.1073/pnas.0409768102</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=555033\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=555033\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Diehn M (2003).
&ldquo;SOURCE: a unified genomic resource of functional annotations, ontologies, and gene expression data.&rdquo;
<EM>Nucleic Acids Research</EM>, <B>31</B>(1), pp. 219&ndash;223.
ISSN 13624962, <a href="http://dx.doi.org/10.1093/nar/gkg014">http://dx.doi.org/10.1093/nar/gkg014</a>, <a href="http://www.nar.oupjournals.org/cgi/doi/10.1093/nar/gkg014">http://www.nar.oupjournals.org/cgi/doi/10.1093/nar/gkg014</a>.</li>
   		<li><p>Draghici S, Khatri P, Eklund AC and Szallasi Z (2006).
&ldquo;Reliability and reproducibility issues in DNA microarray measurements.&rdquo;
<EM>Trends in genetics : TIG</EM>, <B>22</B>(2), pp. 101&ndash;9.
ISSN 0168-9525, <a href="http://dx.doi.org/10.1016/j.tig.2005.12.005">http://dx.doi.org/10.1016/j.tig.2005.12.005</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16380191">http://www.ncbi.nlm.nih.gov/pubmed/16380191</a>.</li>
   		<li><p>Du P, Kibbe Wa and Lin SM (2007).
&ldquo;nuID: a universal naming scheme of oligonucleotides for illumina, affymetrix, and other microarrays.&rdquo;
<EM>Biology direct</EM>, <B>2</B>, pp. 16.
ISSN 1745-6150, <a href="http://dx.doi.org/10.1186/1745-6150-2-16">http://dx.doi.org/10.1186/1745-6150-2-16</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1891274\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1891274\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Epstein MA, Achong BG, Barr YM, Zajac B, Henle G and Henle W (1966).
&ldquo;Morphological and virological investigations on cultured Burkitt tumor lymphoblasts (strain Raji).&rdquo;
<EM>Journal of the National Cancer Institute</EM>, <B>37</B>(4), pp. 547&ndash;59.
ISSN 0027-8874, <a href="http://www.ncbi.nlm.nih.gov/pubmed/4288580">http://www.ncbi.nlm.nih.gov/pubmed/4288580</a>.</li>
   		<li><p>Erkkila T, Lehmusvaara S, Ruusuvuori P, Visakorpi T, Shmulevich I and Lahdesmaki H (2010).
&ldquo;Probabilistic analysis of gene expression measurements from heterogeneous tissues.&rdquo;
<EM>Bioinformatics (Oxford, England)</EM>, <B>26</B>(20), pp. 2571&ndash;7.
ISSN 1367-4811, <a href="http://dx.doi.org/10.1093/bioinformatics/btq406">http://dx.doi.org/10.1093/bioinformatics/btq406</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/20631160 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2951082\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.ncbi.nlm.nih.gov/pubmed/20631160 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2951082\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Fahey JL, Buell DN and Sox HC (1971).
&ldquo;Proliferation and differentiation of lymphoid cells: studies with human lymphoid cell lines and immunoglobulin synthesis.&rdquo;
<EM>Annals of the New York Academy of Sciences</EM>, <B>190</B>, pp. 221&ndash;34.
ISSN 0077-8923, <a href="http://www.ncbi.nlm.nih.gov/pubmed/5290016">http://www.ncbi.nlm.nih.gov/pubmed/5290016</a>.</li>
   		<li><p>Ferreira R, Ohneda K, Yamamoto M and Philipsen S (2005).
&ldquo;GATA1 function, a paradigm for transcription factors in hematopoiesis.&rdquo;
<EM>Molecular and cellular biology</EM>, <B>25</B>(4), pp. 1215.
<a href="http://dx.doi.org/10.1128/MCB.25.4.1215">http://dx.doi.org/10.1128/MCB.25.4.1215</a>, <a href="http://mcb.asm.org/cgi/content/abstract/25/4/1215">http://mcb.asm.org/cgi/content/abstract/25/4/1215</a>.</li>
   		<li><p>Gardy JL, Lynn DJ, Brinkman FSL and Hancock REW (2009).
&ldquo;Enabling a systems biology approach to immunology: focus on innate immunity.&rdquo;
<EM>Trends in immunology</EM>, <B>30</B>(6), pp. 249&ndash;62.
ISSN 1471-4981, <a href="http://dx.doi.org/10.1016/j.it.2009.03.009">http://dx.doi.org/10.1016/j.it.2009.03.009</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/19428301">http://www.ncbi.nlm.nih.gov/pubmed/19428301</a>.</li>
   		<li><p>Gaujoux R and Seoighe C (2011).
&ldquo;Semi-supervised Nonnegative Matrix Factorization for gene expression deconvolution: A case study.&rdquo;
<EM>Infection, genetics and evolution : journal of molecular epidemiology and evolutionary genetics in infectious diseases</EM>.
ISSN 1567-7257, <a href="http://dx.doi.org/10.1016/j.meegid.2011.08.014">http://dx.doi.org/10.1016/j.meegid.2011.08.014</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/21930246">http://www.ncbi.nlm.nih.gov/pubmed/21930246</a>.</li>
   		<li><p>Gaujoux R and Seoighe C (2012).
&ldquo;CellMix: A Comprehensive Framework for Gene Expression Deconvolution.&rdquo;
<EM>submitted</EM>.</li>
   		<li><p>Gaujoux R and Seoighe C (2010).
&ldquo;A flexible R package for nonnegative matrix factorization.&rdquo;
<EM>BMC bioinformatics</EM>, <B>11</B>, pp. 367.
ISSN 1471-2105, <a href="http://dx.doi.org/10.1186/1471-2105-11-367">http://dx.doi.org/10.1186/1471-2105-11-367</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2912887/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2912887/</a>.</li>
   		<li><p>Gellert P, Jenniches K, Braun T and Uchida S (2010).
&ldquo;C-It: a knowledge database for tissue-enriched genes.&rdquo;
<EM>Bioinformatics (Oxford, England)</EM>, <B>26</B>(18), pp. 2328&ndash;33.
ISSN 1367-4811, <a href="http://dx.doi.org/10.1093/bioinformatics/btq417">http://dx.doi.org/10.1093/bioinformatics/btq417</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/20628071">http://www.ncbi.nlm.nih.gov/pubmed/20628071</a>.</li>
   		<li><p>Gentleman RC, Carey VJ, Bates DM, Bolstad B, Dettling M, Dudoit S, Ellis B, Gautier L, Ge Y, Gentry J, Hornik K, Hothorn T, Huber W, Iacus S, Irizarry R, Leisch F, Li C, Maechler M, Rossini AJ, Sawitzki G, Smith C, Smyth G, Tierney L, Yang JYH and Zhang J (2004).
&ldquo;Bioconductor: open software development for computational biology and bioinformatics.&rdquo;
<EM>Genome biology</EM>, <B>5</B>(10), pp. R80.
ISSN 1465-6914, <a href="http://dx.doi.org/10.1186/gb-2004-5-10-r80">http://dx.doi.org/10.1186/gb-2004-5-10-r80</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/15461798">http://www.ncbi.nlm.nih.gov/pubmed/15461798</a>.</li>
   		<li><p>Gong T, Hartmann N, Kohane IS, Brinkmann V, Staedtler F, Letzkus M, Bongiovanni S and Szustakowski JD (2011).
&ldquo;Optimal deconvolution of transcriptional profiling data using quadratic programming with application to complex clinical blood samples.&rdquo;
<EM>PloS one</EM>, <B>6</B>(11), pp. e27156.
ISSN 1932-6203, <a href="http://dx.doi.org/10.1371/journal.pone.0027156">http://dx.doi.org/10.1371/journal.pone.0027156</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3217948\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3217948\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Grcar J (2003).
&ldquo;Optimal sensitivity analysis of linear least squares.&rdquo;
<EM>Lawrence Berkeley National Laboratory, Report LBNL-</EM>, <B>99</B>.
<a href="http://scholar.google.com/scholar?hl=en\&amp;btnG=Search\&amp;q=intitle:Optimal+Sensitivity+Analysis+of+Linear+Least+Squares\#0">http://scholar.google.com/scholar?hl=en\&amp;btnG=Search\&amp;q=intitle:Optimal+Sensitivity+Analysis+of+Linear+Least+Squares\#0</a>.</li>
   		<li><p>Gremse M, Chang A, Schomburg I, Grote A, Scheer M, Ebeling C and Schomburg D (2011).
&ldquo;The BRENDA Tissue Ontology (BTO): the first all-integrating ontology of all organisms for enzyme sources.&rdquo;
<EM>Nucleic acids research</EM>, <B>39</B>(Database issue), pp. D507&ndash;13.
ISSN 1362-4962, <a href="http://dx.doi.org/10.1093/nar/gkq968">http://dx.doi.org/10.1093/nar/gkq968</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3013802\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3013802\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Grigoryev Ya, Kurian SM, Avnur Z, Borie D, Deng J, Campbell D, Sung J, Nikolcheva T, Quinn A, Schulman H, Peng SL, Schaffer R, Fisher J, Mondala T, Head S, Flechner SM, Kantor AB, Marsh C and Salomon DR (2010).
&ldquo;Deconvoluting post-transplant immunity: cell subset-specific mapping reveals pathways for activation and expansion of memory T, monocytes and B cells.&rdquo;
<EM>PloS one</EM>, <B>5</B>(10), pp. e13358.
ISSN 1932-6203, <a href="http://dx.doi.org/10.1371/journal.pone.0013358">http://dx.doi.org/10.1371/journal.pone.0013358</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2954794\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2954794\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Haskell K and Hanson R (1981).
&ldquo;An algorithm for linear least squares problems with equality and nonnegativity constraints.&rdquo;
<EM>Mathematical Programming</EM>, <B>21</B>(1), pp. 98&ndash;118.
<a href="http://www.springerlink.com/index/U677717422346122.pdf">http://www.springerlink.com/index/U677717422346122.pdf</a>.</li>
   		<li><p>Higham N (1996).
<EM>Accuracy and stability of numerical algorithms</EM>,  number 48.
Siam.
<a href="http://books.google.com/books?hl=en\&amp;amp;lr=\&amp;amp;id=EbBWmmPcjPIC\&amp;amp;oi=fnd\&amp;amp;pg=PR13\&amp;amp;dq=Accuracy+and+stability+of+numerical+algorithms\&amp;amp;ots=\_yDHlSI6-t\&amp;amp;sig=G2w4Qr6z2f2pg7Tp1c9R9ABWzEg">http://books.google.com/books?hl=en\&amp;amp;lr=\&amp;amp;id=EbBWmmPcjPIC\&amp;amp;oi=fnd\&amp;amp;pg=PR13\&amp;amp;dq=Accuracy+and+stability+of+numerical+algorithms\&amp;amp;ots=\_yDHlSI6-t\&amp;amp;sig=G2w4Qr6z2f2pg7Tp1c9R9ABWzEg</a>.</li>
   		<li><p>Hoyer P (2004).
&ldquo;Non-negative matrix factorization with sparseness constraints.&rdquo;
<EM>The Journal of Machine Learning Research</EM>, <B>5</B>, pp. 1457&ndash;1469.
<a href="http://portal.acm.org/citation.cfm?id=1044709">http://portal.acm.org/citation.cfm?id=1044709</a>.</li>
   		<li><p>Jacobsen M, Repsilber D, Gutschmidt A, Neher A, Feldmann K, Mollenkopf H, Kaufmann S and Ziegler A (2006).
&ldquo;Deconfounding microarray analysis.&rdquo;
<EM>Methods of information in medicine</EM>, <B>45</B>(5), pp. 557&ndash;563.
<a href="http://www.schattauer.de/de/magazine/uebersicht/zeitschriften-a-z/methods/contents/archive/issue/680/manuscript/7198.html">http://www.schattauer.de/de/magazine/uebersicht/zeitschriften-a-z/methods/contents/archive/issue/680/manuscript/7198.html</a>.</li>
   		<li><p>Janeway CA, Travers P, Walport M and Shlomchik MJ (2001).
<EM>Immunobiology : the immune system in health and disease</EM>, volume 6, 5th edition.
Garland Science.
ISBN 0815341016, <a href="http://www.ncbi.nlm.nih.gov/books/NBK27092/\#A39">http://www.ncbi.nlm.nih.gov/books/NBK27092/\#A39</a>.</li>
   		<li><p>Kim H and Park H (2007).
&ldquo;Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis.&rdquo;
<EM>Bioinformatics (Oxford, England)</EM>, <B>23</B>(12), pp. 1495&ndash;502.
ISSN 1460-2059, <a href="http://dx.doi.org/10.1093/bioinformatics/btm134">http://dx.doi.org/10.1093/bioinformatics/btm134</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/17483501">http://www.ncbi.nlm.nih.gov/pubmed/17483501</a>.</li>
   		<li><p>Kogenaru S, Val C, Hotz-Wagenblatt A and Glatting K (2009).
&ldquo;TissueDistributionDBs: a repository of organism-specific tissue-distribution profiles.&rdquo;
<EM>Theoretical Chemistry Accounts</EM>, <B>125</B>(3-6), pp. 651&ndash;658.
ISSN 1432-881X, <a href="http://dx.doi.org/10.1007/s00214-009-0670-5">http://dx.doi.org/10.1007/s00214-009-0670-5</a>, <a href="http://www.springerlink.com/index/10.1007/s00214-009-0670-5 http://www.springerlink.com/index/V03177J8214R3341.pdf">http://www.springerlink.com/index/10.1007/s00214-009-0670-5 http://www.springerlink.com/index/V03177J8214R3341.pdf</a>.</li>
   		<li><p>Kuhn A, Thu D, Waldvogel HJ, Faull RLM and Luthi-Carter R (2011).
&ldquo;Population-specific expression analysis (PSEA) reveals molecular changes in diseased brain.&rdquo;
<EM>Nature methods</EM>, <B>8</B>(11), pp. 945&ndash;7.
ISSN 1548-7105, <a href="http://dx.doi.org/10.1038/nmeth.1710">http://dx.doi.org/10.1038/nmeth.1710</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/21983921">http://www.ncbi.nlm.nih.gov/pubmed/21983921</a>.</li>
   		<li><p>Lähdesmäki H, Shmulevich L, Dunmire V, Yli-Harja O and Zhang W (2005).
&ldquo;In silico microdissection of microarray data from heterogeneous cell populations.&rdquo;
<EM>BMC bioinformatics</EM>, <B>6</B>, pp. 54.
ISSN 1471-2105, <a href="http://dx.doi.org/10.1186/1471-2105-6-54">http://dx.doi.org/10.1186/1471-2105-6-54</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/15766384">http://www.ncbi.nlm.nih.gov/pubmed/15766384</a>.</li>
   		<li><p>Langfelder P and Horvath S (2008).
&ldquo;WGCNA : an R package for weighted correlation network analysis.&rdquo;
<EM>BMC Bioinformatics</EM>, <B>13</B>.
<a href="http://dx.doi.org/10.1186/1471-2105-9-559">http://dx.doi.org/10.1186/1471-2105-9-559</a>.</li>
   		<li><p>Lee DD and Seung H (2001).
&ldquo;Algorithms for non-negative matrix factorization.&rdquo;
<EM>Advances in neural information processing systems</EM>.
<a href="http://scholar.google.com/scholar?q=intitle:Algorithms+for+non-negative+matrix+factorization\#0">http://scholar.google.com/scholar?q=intitle:Algorithms+for+non-negative+matrix+factorization\#0</a>.</li>
   		<li><p>Lee DD and Seung HS (1999).
&ldquo;Learning the parts of objects by non-negative matrix factorization.&rdquo;
<EM>Nature</EM>, <B>401</B>(6755), pp. 788&ndash;91.
ISSN 0028-0836, <a href="http://dx.doi.org/10.1038/44565">http://dx.doi.org/10.1038/44565</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/10548103">http://www.ncbi.nlm.nih.gov/pubmed/10548103</a>.</li>
   		<li><p>Liang S, Li Y, Be X, Howes S and Liu W (2006).
&ldquo;Detecting and profiling tissue-selective genes.&rdquo;
<EM>Physiological genomics</EM>, <B>26</B>(2), pp. 158&ndash;62.
ISSN 1531-2267, <a href="http://dx.doi.org/10.1152/physiolgenomics.00313.2005">http://dx.doi.org/10.1152/physiolgenomics.00313.2005</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16684803">http://www.ncbi.nlm.nih.gov/pubmed/16684803</a>.</li>
   		<li><p>Liu W, Yuan K and Ye D (2008).
&ldquo;Reducing microarray data via nonnegative matrix factorization for visualization and clustering analysis.&rdquo;
<EM>Journal of biomedical informatics</EM>, <B>41</B>(4), pp. 602&ndash;6.
ISSN 1532-0480, <a href="http://dx.doi.org/10.1016/j.jbi.2007.12.003">http://dx.doi.org/10.1016/j.jbi.2007.12.003</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/18234564">http://www.ncbi.nlm.nih.gov/pubmed/18234564</a>.</li>
   		<li><p>Liu X, Yu X, Zack DJ, Zhu H and Qian J (2008).
&ldquo;TiGER: a database for tissue-specific gene expression and regulation.&rdquo;
<EM>BMC bioinformatics</EM>, <B>9</B>, pp. 271.
ISSN 1471-2105, <a href="http://dx.doi.org/10.1186/1471-2105-9-271">http://dx.doi.org/10.1186/1471-2105-9-271</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2438328\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2438328\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Lottaz C, Yang X, Scheid S and Spang R (2006).
&ldquo;OrderedList&ndash;a bioconductor package for detecting similarity in ordered gene lists.&rdquo;
<EM>Bioinformatics (Oxford, England)</EM>, <B>22</B>(18), pp. 2315&ndash;6.
ISSN 1367-4811, <a href="http://dx.doi.org/10.1093/bioinformatics/btl385">http://dx.doi.org/10.1093/bioinformatics/btl385</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16844712">http://www.ncbi.nlm.nih.gov/pubmed/16844712</a>.</li>
   		<li><p>Lu P, Nakorchevskiy A and Marcotte EM (2003).
&ldquo;Expression deconvolution: a reinterpretation of DNA microarray data reveals dynamic changes in cell populations.&rdquo;
<EM>Proceedings of the National Academy of Sciences of the United States of America</EM>, <B>100</B>(18), pp. 10370&ndash;5.
ISSN 0027-8424, <a href="http://dx.doi.org/10.1073/pnas.1832361100">http://dx.doi.org/10.1073/pnas.1832361100</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=193568\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=193568\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Mecham BH, Klus GT, Strovel J, Augustus M, Byrne D, Bozso P, Wetmore DZ, Mariani TJ, Kohane IS and Szallasi Z (2004).
&ldquo;Sequence-matched probes produce increased cross-platform consistency and more reproducible biological results in microarray-based gene expression measurements.&rdquo;
<EM>Nucleic acids research</EM>, <B>32</B>(9), pp. e74.
ISSN 1362-4962, <a href="http://dx.doi.org/10.1093/nar/gnh071">http://dx.doi.org/10.1093/nar/gnh071</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=419626\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=419626\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Miller Ja, Cai C, Langfelder P, Geschwind DH, Kurian SM, Salomon DR and Horvath S (2011).
&ldquo;Strategies for aggregating gene expression data: The collapseRows R function.&rdquo;
<EM>BMC Bioinformatics</EM>, <B>12</B>(1), pp. 322.
ISSN 1471-2105, <a href="http://dx.doi.org/10.1186/1471-2105-12-322">http://dx.doi.org/10.1186/1471-2105-12-322</a>, <a href="http://www.biomedcentral.com/1471-2105/12/322">http://www.biomedcentral.com/1471-2105/12/322</a>.</li>
   		<li><p>Novershtern N, Subramanian A, Lawton LN, Mak RH, Haining WN, McConkey ME, Habib N, Yosef N, Chang CY, Shay T, Frampton GM, Drake ACB, Leskov I, Nilsson B, Preffer F, Dombkowski D, Evans JW, Liefeld T, Smutko JS, Chen J, Friedman N, Young Ra, Golub TR, Regev A and Ebert BL (2011).
&ldquo;Densely interconnected transcriptional circuits control cell states in human hematopoiesis.&rdquo;
<EM>Cell</EM>, <B>144</B>(2), pp. 296&ndash;309.
ISSN 1097-4172, <a href="http://dx.doi.org/10.1016/j.cell.2011.01.004">http://dx.doi.org/10.1016/j.cell.2011.01.004</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3049864\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3049864\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Paatero P and Tapper U (1994).
&ldquo;Positive matrix factorization: A non-negative factor model with optimal utilization of error estimates of data values.&rdquo;
<EM>Environmetrics</EM>, <B>5</B>(2), pp. 111&ndash;126.
<a href="http://dx.doi.org/10.1002/env.3170050203">http://dx.doi.org/10.1002/env.3170050203</a>, <a href="http://www3.interscience.wiley.com/cgi-bin/abstract/113468839/ABSTRACT">http://www3.interscience.wiley.com/cgi-bin/abstract/113468839/ABSTRACT</a>.</li>
   		<li><p>Palmer C, Diehn M, Alizadeh Aa and Brown PO (2006).
&ldquo;Cell-type specific gene expression profiles of leukocytes in human peripheral blood.&rdquo;
<EM>BMC genomics</EM>, <B>7</B>, pp. 115.
ISSN 1471-2164, <a href="http://dx.doi.org/10.1186/1471-2164-7-115">http://dx.doi.org/10.1186/1471-2164-7-115</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16704732">http://www.ncbi.nlm.nih.gov/pubmed/16704732</a>.</li>
   		<li><p>Parkinson H, Kapushesky M, Kolesnikov N, Rustici G, Shojatalab M, Abeygunawardena N, Berube H, Dylag M, Emam I, Farne A, Holloway E, Lukk M, Malone J, Mani R, Pilicheva E, Rayner TF, Rezwan F, Sharma A, Williams E, Bradley XZ, Adamusiak T, Brandizi M, Burdett T, Coulson R, Krestyaninova M, Kurnosov P, Maguire E, Neogi SG, Rocca-Serra P, Sansone S, Sklyar N, Zhao M, Sarkans U and Brazma A (2009).
&ldquo;ArrayExpress update&ndash;from an archive of functional genomics experiments to the atlas of gene expression.&rdquo;
<EM>Nucleic acids research</EM>, <B>37</B>(Database issue), pp. D868&ndash;72.
ISSN 1362-4962, <a href="http://dx.doi.org/10.1093/nar/gkn889">http://dx.doi.org/10.1093/nar/gkn889</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2686529\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2686529\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Patocs A, Zhang L, Xu Y, Weber F, Caldes T, Mutter G, Platzer P and Eng C (2007).
&ldquo;Breast-cancer stromal cells with TP53 mutations and nodal metastases.&rdquo;
<EM>New England Journal of Medicine</EM>, <B>357</B>(25), pp. 2543&ndash;2551.
<a href="http://www.nejm.org/doi/full/10.1056/NEJMoa071825">http://www.nejm.org/doi/full/10.1056/NEJMoa071825</a>.</li>
   		<li><p>R Development Core Team (2011).
<EM>R: A Language and Environment for Statistical Computing</EM>.
R Foundation for Statistical Computing, Vienna, Austria.
<a href="http://www.r-project.org">http://www.r-project.org</a>.</li>
   		<li><p>Repsilber D, Kern S, Telaar A, Walzl G, Black GF, Selbig J, Parida SK, Kaufmann SHE and Jacobsen M (2010).
&ldquo;Biomarker discovery in heterogeneous tissue samples -taking the in-silico deconfounding approach.&rdquo;
<EM>BMC bioinformatics</EM>, <B>11</B>, pp. 27.
ISSN 1471-2105, <a href="http://dx.doi.org/10.1186/1471-2105-11-27">http://dx.doi.org/10.1186/1471-2105-11-27</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/20070912">http://www.ncbi.nlm.nih.gov/pubmed/20070912</a>.</li>
   		<li><p>Roy S, Lane T, Allen C, Aragon AD and Werner-Washburne M (2006).
&ldquo;A hidden-state Markov model for cell population deconvolution.&rdquo;
<EM>Journal of computational biology : a journal of computational molecular cell biology</EM>, <B>13</B>(10), pp. 1749&ndash;74.
ISSN 1066-5277, <a href="http://dx.doi.org/10.1089/cmb.2006.13.1749">http://dx.doi.org/10.1089/cmb.2006.13.1749</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/17238843">http://www.ncbi.nlm.nih.gov/pubmed/17238843</a>.</li>
   		<li><p>Schneider S, Smith T and Hansen U (2011).
&ldquo;SCOREM: statistical consolidation of redundant expression measures.&rdquo;
<EM>Nucleic acids research</EM>, pp. 1&ndash;12.
ISSN 1362-4962, <a href="http://dx.doi.org/10.1093/nar/gkr1270">http://dx.doi.org/10.1093/nar/gkr1270</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/22210887">http://www.ncbi.nlm.nih.gov/pubmed/22210887</a>.</li>
   		<li><p>Schneider U, Schwenk HU and Bornkamm G (1977).
&ldquo;Characterization of EBV-genome negative &quot;null&quot; and &quot;T&quot; cell lines derived from children with acute lymphoblastic leukemia and leukemic transformed non-Hodgkin lymphoma.&rdquo;
<EM>International journal of cancer. Journal international du cancer</EM>, <B>19</B>(5), pp. 621&ndash;6.
ISSN 0020-7136, <a href="http://www.ncbi.nlm.nih.gov/pubmed/68013">http://www.ncbi.nlm.nih.gov/pubmed/68013</a>.</li>
   		<li><p>Shen-Orr SS, Tibshirani R, Khatri P, Bodian DL, Staedtler F, Perry NM, Hastie T, Sarwal MM, Davis MM and Butte AJ (2010).
&ldquo;Cell type-specific gene expression differences in complex tissues.&rdquo;
<EM>Nature methods</EM>, <B>7</B>(4), pp. 287&ndash;9.
ISSN 1548-7105, <a href="http://dx.doi.org/10.1038/nmeth.1439">http://dx.doi.org/10.1038/nmeth.1439</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/20208531">http://www.ncbi.nlm.nih.gov/pubmed/20208531</a>.</li>
   		<li><p>Shi L, Reid LH, Jones WD, Shippy R, Warrington JA, Baker SC, Collins PJ, Longueville Fd, Kawasaki ES, Lee KY, Luo Y, Sun YA, Willey JC, Setterquist RA, Fischer GM, Tong W, Dragan YP, Dix DJ, Frueh FW, Goodsaid FM, Herman D, Jensen RV, Johnson CD, Lobenhofer EK, Puri RK, Schrf U, Thierry-Mieg J, Wang C, Wilson M, Wolber PK, Zhang L, Amur S, Bao W, Barbacioru CC, Lucas AB, Bertholet V, Boysen C, Bromley B, Brown D, Brunner A, Canales R, Cao XM, Cebula TA, Chen JJ, Cheng J, Chu T, Chudin E, Corson J, Corton JC, Croner LJ, Davies C, Davison TS, Delenstarr G, Deng X, Dorris D, Eklund AC, Fan X, Fang H, Fulmer-Smentek S, Fuscoe JC, Gallagher K, Ge W, Guo L, Guo X, Hager J, Haje PK, Han J, Han T, Harbottle HC, Harris SC, Hatchwell E, Hauser CA, Hester S, Hong H, Hurban P, Jackson SA, Ji H, Knight CR, Kuo WP, LeClerc JE, Levy S, Li Q, Liu C, Liu Y, Lombardi MJ, Ma Y, Magnuson SR, Maqsodi B, McDaniel T, Mei N, Myklebost O, Ning B, Novoradovskaya N, Orr MS, Osborn TW, Papallo A, Patterson TA, Perkins RG, Peters EH, Peterson R, Philips KL, Pine PS, Pusztai L, Qian F, Ren H, Rosen M, Rosenzweig BA, Samaha RR, Schena M, Schroth GP, Shchegrova S, Smith DD, Staedtler F, Su Z, Sun H, Szallasi Z, Tezak Z, Thierry-Mieg D, Thompson KL, Tikhonova I, Turpaz Y, Vallanat B, Van C, Walker SJ, Wang SJ, Wang Y, Wolfinger R, Wong A, Wu J, Xiao C, Xie Q, Xu J, Yang W, Zhang L, Zhong S, Zong Y and Slikker W (2006).
&ldquo;The MicroArray Quality Control (MAQC) project shows inter- and intraplatform reproducibility of gene expression measurements.&rdquo;
<EM>Nature biotechnology</EM>, <B>24</B>(9), pp. 1151&ndash;61.
ISSN 1087-0156, <a href="http://dx.doi.org/10.1038/nbt1239">http://dx.doi.org/10.1038/nbt1239</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16964229">http://www.ncbi.nlm.nih.gov/pubmed/16964229</a>.</li>
   		<li><p>Siegert S, Cabuy E, Scherf BG, Kohler H, Panda S, Le Y, Fehling HJ, Gaidatzis D, Stadler MB and Roska B (2012).
&ldquo;Transcriptional code and disease map for adult retinal cell types.&rdquo;
<EM>Nature neuroscience</EM>.
ISSN 1546-1726, <a href="http://dx.doi.org/10.1038/nn.3032">http://dx.doi.org/10.1038/nn.3032</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/22267162">http://www.ncbi.nlm.nih.gov/pubmed/22267162</a>.</li>
   		<li><p>Smyth GK (2004).
&ldquo;Linear models and empirical bayes methods for assessing differential expression in microarray experiments.&rdquo;
<EM>Statistical applications in genetics and molecular biology</EM>, <B>3</B>(1), pp. Article3.
ISSN 1544-6115, <a href="http://dx.doi.org/10.2202/1544-6115.1027">http://dx.doi.org/10.2202/1544-6115.1027</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16646809">http://www.ncbi.nlm.nih.gov/pubmed/16646809</a>.</li>
   		<li><p>Stuart RO, Wachsman W, Berry CC, Wang-Rodriguez J, Wasserman L, Klacansky I, Masys D, Arden K, Goodison S, McClelland M, Wang Y, Sawyers A, Kalcheva I, Tarin D and Mercola D (2004).
&ldquo;In silico dissection of cell-type-associated patterns of gene expression in prostate cancer.&rdquo;
<EM>Proceedings of the National Academy of Sciences of the United States of America</EM>, <B>101</B>(2), pp. 615&ndash;20.
ISSN 0027-8424, <a href="http://dx.doi.org/10.1073/pnas.2536479100">http://dx.doi.org/10.1073/pnas.2536479100</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=327196&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=327196&amp;tool=pmcentrez&amp;rendertype=abstract</a>.</li>
   		<li><p>Su AI, Wiltshire T, Batalov S, Lapp H, Ching Ka, Block D, Zhang J, Soden R, Hayakawa M, Kreiman G, Cooke MP, Walker JR and Hogenesch JB (2004).
&ldquo;A gene atlas of the mouse and human protein-encoding transcriptomes.&rdquo;
<EM>Proceedings of the National Academy of Sciences of the United States of America</EM>, <B>101</B>(16), pp. 6062&ndash;7.
ISSN 0027-8424, <a href="http://dx.doi.org/10.1073/pnas.0400782101">http://dx.doi.org/10.1073/pnas.0400782101</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=395923\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=395923\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette Ma, Paulovich A, Pomeroy SL, Golub TR, Lander ES and Mesirov JP (2005).
&ldquo;Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.&rdquo;
<EM>Proceedings of the National Academy of Sciences of the United States of America</EM>, <B>102</B>(43), pp. 15545&ndash;50.
ISSN 0027-8424, <a href="http://dx.doi.org/10.1073/pnas.0506580102">http://dx.doi.org/10.1073/pnas.0506580102</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16199517">http://www.ncbi.nlm.nih.gov/pubmed/16199517</a>.</li>
   		<li><p>Tsuchiya S, Yamabe M, Yamaguchi Y, Kobayashi Y, Konno T and Tada K (1980).
&ldquo;Establishment and characterization of a human acute monocytic leukemia cell line (THP-1).&rdquo;
<EM>International journal of cancer. Journal international du cancer</EM>, <B>26</B>(2), pp. 171&ndash;6.
ISSN 0020-7136, <a href="http://www.ncbi.nlm.nih.gov/pubmed/6970727">http://www.ncbi.nlm.nih.gov/pubmed/6970727</a>.</li>
   		<li><p>Tusher VG, Tibshirani R and Chu G (2001).
&ldquo;Significance analysis of microarrays applied to the ionizing radiation response.&rdquo;
<EM>Proceedings of the National Academy of Sciences of the United States of America</EM>, <B>98</B>(9), pp. 5116&ndash;21.
ISSN 0027-8424, <a href="http://dx.doi.org/10.1073/pnas.091062498">http://dx.doi.org/10.1073/pnas.091062498</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/11309499">http://www.ncbi.nlm.nih.gov/pubmed/11309499</a>.</li>
   		<li><p>Van Benthem M and Keenan MR (2004).
&ldquo;Fast algorithm for the solution of large-scale non-negativity-constrained least squares problems.&rdquo;
<EM>Journal of Chemometrics</EM>, <B>18</B>(10), pp. 441&ndash;450.
ISSN 0886-9383, <a href="http://dx.doi.org/10.1002/cem.889">http://dx.doi.org/10.1002/cem.889</a>, <a href="http://doi.wiley.com/10.1002/cem.889">http://doi.wiley.com/10.1002/cem.889</a>.</li>
   		<li><p>Van den Meersche K, Soetaert K and Van Oevelen D (2009).
&ldquo;xsample (): An R function for sampling linear inverse problems.&rdquo;
<EM>Journal of Statistical Software</EM>, <B>30</B>(Code Snippet 1), pp. 1&ndash;15.
<a href="http://biblio.ugent.be/input/download?func=downloadFile\&amp;amp;fileOId=677001">http://biblio.ugent.be/input/download?func=downloadFile\&amp;amp;fileOId=677001</a>.</li>
   		<li><p>Venet D, Pecasse F, Maenhaut C and Bersini H (2001).
&ldquo;Separation of samples into their constituents using gene expression data.&rdquo;
<EM>Bioinformatics</EM>, <B>17</B>(suppl 1), pp. S279.
<a href="http://bioinformatics.oxfordjournals.org/content/17/suppl\_1/S279.short">http://bioinformatics.oxfordjournals.org/content/17/suppl\_1/S279.short</a>.</li>
   		<li><p>Wang M, Master SR and Chodosh La (2006).
&ldquo;Computational expression deconvolution in a complex mammalian organ.&rdquo;
<EM>BMC bioinformatics</EM>, <B>7</B>, pp. 328.
ISSN 1471-2105, <a href="http://dx.doi.org/10.1186/1471-2105-7-328">http://dx.doi.org/10.1186/1471-2105-7-328</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16817968">http://www.ncbi.nlm.nih.gov/pubmed/16817968</a>.</li>
   		<li><p>Watkins Na, Gusnanto A, Bono Bd, De S, Miranda-Saavedra D, Hardie DL, Angenent WGJ, Attwood AP, Ellis PD, Erber W, Foad NS, Garner SF, Isacke CM, Jolley J, Koch K, Macaulay IC, Morley SL, Rendon A, Rice KM, Taylor N, Thijssen-Timmer DC, Tijssen MR, Schoot CEvd, Wernisch L, Winzer T, Dudbridge F, Buckley CD, Langford CF, Teichmann S, Gottgens B and Ouwehand WH (2009).
&ldquo;A HaemAtlas: characterizing gene expression in differentiated human blood cells.&rdquo;
<EM>Blood</EM>, <B>113</B>(19), pp. e1&ndash;9.
ISSN 1528-0020, <a href="http://dx.doi.org/10.1182/blood-2008-06-162958">http://dx.doi.org/10.1182/blood-2008-06-162958</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2680378\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2680378\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Whitney A, Diehn M, Popper S, Alizadeh A, Boldrick J, Relman D and Brown P (2003).
&ldquo;Individuality and variation in gene expression patterns in human blood.&rdquo;
<EM>Proceedings of the National Academy of Sciences of the United States of America</EM>, <B>100</B>(4), pp. 1896.
<a href="http://www.pnas.org/content/100/4/1896.short">http://www.pnas.org/content/100/4/1896.short</a>.</li>
   		<li><p>Xiao S, Zhang C, Zou Q and Ji Z (2010).
&ldquo;TiSGeD: a database for tissue-specific genes.&rdquo;
<EM>Bioinformatics (Oxford, England)</EM>, <B>26</B>(9), pp. 1273&ndash;5.
ISSN 1367-4811, <a href="http://dx.doi.org/10.1093/bioinformatics/btq109">http://dx.doi.org/10.1093/bioinformatics/btq109</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2859128\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2859128\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Yang X and Sun X (2007).
&ldquo;Meta-analysis of several gene lists for distinct types of cancer: a simple way to reveal common prognostic markers.&rdquo;
<EM>BMC bioinformatics</EM>, <B>8</B>, pp. 118.
ISSN 1471-2105, <a href="http://dx.doi.org/10.1186/1471-2105-8-118">http://dx.doi.org/10.1186/1471-2105-8-118</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1853113\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1853113\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Yang X, Ye Y, Wang G, Huang H, Yu D and Liang S (2011).
&ldquo;VeryGene: linking tissue-specific genes to diseases, drugs, and beyond for knowledge discovery.&rdquo;
<EM>Physiological genomics</EM>, <B>43</B>(8), pp. 457&ndash;60.
ISSN 1531-2267, <a href="http://dx.doi.org/10.1152/physiolgenomics.00178.2010">http://dx.doi.org/10.1152/physiolgenomics.00178.2010</a>, <a href="http://physiolgenomics.physiology.org/cgi/content/abstract/43/8/457">http://physiolgenomics.physiology.org/cgi/content/abstract/43/8/457</a>.</li>
   		<li><p>Zhao Y and Simon R (2010).
&ldquo;Gene expression deconvolution in clinical samples.&rdquo;
<EM>Genome medicine</EM>, <B>2</B>(12), pp. 93.
ISSN 1756-994X, <a href="http://dx.doi.org/10.1186/gm214">http://dx.doi.org/10.1186/gm214</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3025435\&amp;tool=pmcentrez\&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3025435\&amp;tool=pmcentrez\&amp;rendertype=abstract</a>.</li>
   		<li><p>Zhong Y, Wan Y, Pang K, Chow LM and Liu Z (2013).
&ldquo;Digital sorting of complex tissues for cell type-specific gene expression profiles.&rdquo;
<EM>BMC Bioinformatics</EM>, <B>14</B>(1), pp. 89.
ISSN 1471-2105, <a href="http://dx.doi.org/10.1186/1471-2105-14-89">http://dx.doi.org/10.1186/1471-2105-14-89</a>, <a href="http://www.biomedcentral.com/1471-2105/14/89">http://www.biomedcentral.com/1471-2105/14/89</a>.</li>
  </ol>
</div></div>
      
      <footer>
      <p class="pull-right"><a href="#">Back to top</a></p>
<p>Built by <a href="https://github.com/hadley/staticdocs">staticdocs</a>. Styled with <a href="http://twitter.github.com/bootstrap">bootstrap</a>.</p>
      </footer>
    </div>
  </body>
</html>