<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
<title>.qprog. CellMix 1.5.1</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="author" content="">

<!-- bootstrap css -->
<link href="css/bootstrap.css" rel="stylesheet">
<link href="css/bootstrap-responsive.css" rel="stylesheet">
<!-- highlighting css -->
<link href="css/highlight.css" rel="stylesheet">
<!-- custom css -->
<link href="css/staticdocs.css" rel="stylesheet">

<!-- bootstrap javascript -->
<script src="js/jquery.min.js"></script>
<script src="js/bootstrap.min.js"></script>
<!-- d3 word cloud -->
<script src="js/d3.v2.min.js"></script>
<script src="js/d3.layout.cloud.js"></script>

<!--[if lt IE 9]>
  <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
<![endif]-->
  </head>

  <body>
    <div class="navbar">
  <div class="navbar-inner">
    <div class = "container">
      <a class="brand" href="index.html">CellMix 1.5.1</a>
      <div class="nav">
        <ul class="nav">
        <li><a href="_MAN.html"><i class="icon-home icon-white"></i> Index</a></li>
       </ul>
   	  </div>
    </div>
  </div>
</div>

    <div class="container">
      <header>
        
      </header>
      
      <h1>Proportion or Signature Estimation by Quadratic Programming</h1>

<div class="row">
  <div class="span8">
    <h2>Usage</h2>
    <pre>.qprog(X, seed, data = NULL, exact = TRUE, log = NULL, ...)</pre>
    
    <h2>Arguments</h2>
    <dl>
      <dt>X</dt>
      <dd>target matrix</dd>
      <dt>seed</dt>
      <dd>initial NMF model as an <code><a href='http://www.inside-r.org/r-doc/NMF/NMF-package'>NMF</a></code>
  object.</dd>
      <dt>data</dt>
      <dd>markers as <code><a href='MarkerList.html'>MarkerList</a></code> object.</dd>
      <dt>exact</dt>
      <dd>logical that specifies if one should impose
  a sum-to-one (<code>TRUE</code>) or sum-to-less-than-one
  (<code>FALSE</code>) constraint on the proportions.</dd>
      <dt>...</dt>
      <dd>extra parameters DSAproportionspassed to
  <code><a href='mlsei.html'>mlsei</a></code>.</dd>
      <dt>log</dt>
      <dd>indicates if the data are in log-scale or
  should be converted to linear-scale. This is relevant
  because the DSA algorithm assumes that the input mixed
  data are in linear scale (i.e. not log-trasnformed).

  If <code>NULL</code>, then data's scale is detected by
  <code>link{is_logscale}</code> and conversion to linear-scale
  is performed if necessary. If <code>TRUE</code> the data is
  exponentialised (using log base 2). If <code>FALSE</code> the
  data is left unchanged (the detected log scale is
  displayed in verbose mode). If a number, then it is used
  as the log base to exponentialise the data.</dd>
    </dl>
    
    <div class="Value">
      <h2>Value</h2>
      
      <p>an <code><a href='http://www.inside-r.org/r-doc/NMF/NMF-package'>NMF</a></code> object.</p>
  
    </div>

    <div class="Description">
      <h2>Description</h2>
      
      <p>This is the workhorse function for the ged algorithms
  &#145;qprog&#146;, &#145;cs-qprog&#146;, and &#145;DSA&#146;.</p>
  
    </div>

    <div class="Details">
      <h2>Details</h2>
      
      <p>The function implements deconvolution methods that use
  quadratic programming techniques to estimate proportions
  from signatures (algorithm
  <code><a href='gedAlgorithm.qprog.html'>qprog</a></code> from <cite>Gong et al.
  (2011)</cite>), or both signatures and proportions from marker
  genes (algorithm <code><a href='gedAlgorithm.DSA.html'>DSA</a></code> from
  <cite>Zhong et al. (2013)</cite>).</p>
  
    </div>

    <div class="References">
      <h2>References</h2>
      
      <p>Gong T, Hartmann N, Kohane IS, Brinkmann V, Staedtler F,
  Letzkus M, Bongiovanni S and Szustakowski JD (2011).
  "Optimal deconvolution of transcriptional profiling data
  using quadratic programming with application to complex
  clinical blood samples." _PloS one_, *6*(11), pp. e27156.
  ISSN 1932-6203, <URL:
  http://dx.doi.org/10.1371/journal.pone.0027156>, <URL:
  http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3217948\&tool=pmcentrez\&rendertype=abstract>.</p>
  
      <p>Zhong Y, Wan Y, Pang K, Chow LM and Liu Z (2013).
  "Digital sorting of complex tissues for cell
  type-specific gene expression profiles." _BMC
  Bioinformatics_, *14*(1), pp. 89. ISSN 1471-2105, <URL:
  http://dx.doi.org/10.1186/1471-2105-14-89>, <URL:
  http://www.biomedcentral.com/1471-2105/14/89>.</p>
  
    </div>
      </div>
  <div class="span4">
    <!-- <ul>
      <li>.qprog</li>
    </ul>
    <ul>
      <li>internal</li>
    </ul> -->
      
        
  </div>
</div>
      
      <footer>
      <p class="pull-right"><a href="#">Back to top</a></p>
<p>Built by <a href="https://github.com/hadley/staticdocs">staticdocs</a>. Styled with <a href="http://twitter.github.com/bootstrap">bootstrap</a>.</p>
      </footer>
    </div>
  </body>
</html>