<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
<title>.mTIGER. CellMix 1.5.1</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="author" content="">

<!-- bootstrap css -->
<link href="css/bootstrap.css" rel="stylesheet">
<link href="css/bootstrap-responsive.css" rel="stylesheet">
<!-- highlighting css -->
<link href="css/highlight.css" rel="stylesheet">
<!-- custom css -->
<link href="css/staticdocs.css" rel="stylesheet">

<!-- bootstrap javascript -->
<script src="js/jquery.min.js"></script>
<script src="js/bootstrap.min.js"></script>
<!-- d3 word cloud -->
<script src="js/d3.v2.min.js"></script>
<script src="js/d3.layout.cloud.js"></script>

<!--[if lt IE 9]>
  <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
<![endif]-->
  </head>

  <body>
    <div class="navbar">
  <div class="navbar-inner">
    <div class = "container">
      <a class="brand" href="index.html">CellMix 1.5.1</a>
      <div class="nav">
        <ul class="nav">
        <li><a href="_MAN.html"><i class="icon-home icon-white"></i> Index</a></li>
       </ul>
   	  </div>
    </div>
  </div>
</div>

    <div class="container">
      <header>
        
      </header>
      
      <h1>TiGER - Human Tissue Specific Genes</h1>

<div class="row">
  <div class="span8">
    <h2>Usage</h2>
    <pre>.mTIGER(score = 5, log10P = 3.47853)</pre>
    
    <h2>Arguments</h2>
    <dl>
      <dt>score</dt>
      <dd>minimum score for a gene to be included in
  the list</dd>
      <dt>log10P</dt>
      <dd>minimum value for -log10(P-value) for a
  gene to be included in the list</dd>
    </dl>
    
    <div class="Description">
      <h2>Description</h2>
      
      <p><span class = "pkg">CellMix</span> access key: &#147;TIGER&#148;</p>
  
      <p>Marker list created from the <code><a href='TIGER.html'>TIGER</a></code> dataset
  (<cite>Liu et al. (2008)</cite>). The TiGER database is based
  on UniGene EST distribution profiles. The markers are
  defined by thresholds on enrichment scores and their
  associated log10 p-values.</p>
  
    </div>

    <div class="Details">
      <h2>Details</h2>
      
      <p>The <code>MarkerList</code> object is generated by the internal
  function <code>.mTIGER</code>, to which arguments can be passed
  via <code>MarkerList('TIGER', ...)</code> to build lists using
  different filtering threshold.</p>
  
      <p>The function <code><a href='convertIDs.html'>convertIDs</a></code> can be used to
  convert the primary identifiers into other identifiers
  (e.g. ENTREZ ids or microarray probe ids), using
  up-to-date mappings from Bioconductor annotation
  packages.</p>
  
    </div>

    <div class="References">
      <h2>References</h2>
      
      <p>Liu X, Yu X, Zack DJ, Zhu H and Qian J (2008). "TiGER: a
  database for tissue-specific gene expression and
  regulation." _BMC bioinformatics_, *9*, pp. 271. ISSN
  1471-2105, <URL:
  http://dx.doi.org/10.1186/1471-2105-9-271>, <URL:
  http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2438328\&tool=pmcentrez\&rendertype=abstract>.</p>
  
    </div>
    
    <h2 id="examples">Examples</h2>
    <pre class="examples"><div class='input'>MarkerList('TIGER')
</div>
<div class='output'><object of class: MarkerList>
Types: bladder, blood, ..., uterus (total: 30)
Mode: numeric
setName: TIGER 
geneIds: Hs.405866, Hs.281295, ..., Hs.424414 (total: 7743)
geneIdType: Unigene (org.Hs.eg.db)
collectionType: Null 
geneValues: 99.8603, 78.4616, ..., 5.04784 
details: use 'details(object)'
</div></pre>
  </div>
  <div class="span4">
    <!-- <ul>
      <li>.mTIGER</li><li>TIGER-markers</li>
    </ul>
    <ul>
      
    </ul> -->
      
        
  </div>
</div>
      
      <footer>
      <p class="pull-right"><a href="#">Back to top</a></p>
<p>Built by <a href="https://github.com/hadley/staticdocs">staticdocs</a>. Styled with <a href="http://twitter.github.com/bootstrap">bootstrap</a>.</p>
      </footer>
    </div>
  </body>
</html>