<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
<title>DSection. CellMix 1.5.1</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="author" content="
  Original Matlab code: Timo Erkkila

  Wrapper function: Renaud Gaujoux
">

<!-- bootstrap css -->
<link href="css/bootstrap.css" rel="stylesheet">
<link href="css/bootstrap-responsive.css" rel="stylesheet">
<!-- highlighting css -->
<link href="css/highlight.css" rel="stylesheet">
<!-- custom css -->
<link href="css/staticdocs.css" rel="stylesheet">

<!-- bootstrap javascript -->
<script src="js/jquery.min.js"></script>
<script src="js/bootstrap.min.js"></script>
<!-- d3 word cloud -->
<script src="js/d3.v2.min.js"></script>
<script src="js/d3.layout.cloud.js"></script>

<!--[if lt IE 9]>
  <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
<![endif]-->
  </head>

  <body>
    <div class="navbar">
  <div class="navbar-inner">
    <div class = "container">
      <a class="brand" href="index.html">CellMix 1.5.1</a>
      <div class="nav">
        <ul class="nav">
        <li><a href="_MAN.html"><i class="icon-home icon-white"></i> Index</a></li>
       </ul>
   	  </div>
    </div>
  </div>
</div>

    <div class="container">
      <header>
        
      </header>
      
      <h1>DSection Gene Expression Deconvolution Method</h1>

<div class="row">
  <div class="span8">
    <h2>Usage</h2>
    <pre>DSection(Y, p0, nSamples, groups = NULL, nBurnIn = 4 * nSamples, W0 = max(nrow(p0), 
  10), W_proposal = 100, samplep = TRUE, summarize = TRUE, verbose = FALSE)</pre>
    
    <h2>Arguments</h2>
    <dl>
      <dt>Y</dt>
      <dd><I-by-J> matrix of measurements from
  heterogeneous tissues. I is the number of
  probes/genes/etc., and J is the number of tissues.</dd>
      <dt>p0</dt>
      <dd><T-by-J> matrix of prior predictions on cell
  type proportions. T is the number of cell types, and
  columns in <code>p0</code> must be positive and add up to one.</dd>
      <dt>groups</dt>
      <dd><1-by-J> vector of treatment indices, so
  that unique(Treatment) = [1,2,...,C], where C is the
  number of treatments including control, i.e., "no
  treatment", if available.</dd>
      <dt>W0</dt>
      <dd>Prior prediction weight, i.e., degree of
  confidence, on <code>p0</code>. Defines the peakedness of
  Dirichlet density around p0. NOTE: keep W0 >= T.</dd>
      <dt>W_proposal</dt>
      <dd>Transition kernel weight, defines the
  peakedness of Dirichlet density around p*, the old value.
  The higher W_proposal is, the smaller the proposal steps
  around p* are.</dd>
      <dt>nBurnIn</dt>
      <dd>Amount of burn-in. NOTE: keep nBurnIn >
  0.</dd>
      <dt>nSamples</dt>
      <dd>Amount of sampling. NOTE: keep nSamples >
  0.</dd>
      <dt>samplep</dt>
      <dd>logical value, indicating whether to
  sample from the posterior for cell type proportions
  (<code>TRUE</code>) or not (<code>FALSE</code>). SUGGESTED USE:
  sample from the posterior (samplep = 1).</dd>
      <dt>summarize</dt>
      <dd>logical indicating whether only average
  values should be returned -- and computed.</dd>
      <dt>verbose</dt>
      <dd>logical that indicates if verbose messages
  should be shown.</dd>
    </dl>
    
    <div class="Source">
      <h2>Source</h2>
      
      <p><a href = 'http://www.cs.tut.fi/~erkkila2/software/dsection/DSection.m'>http://www.cs.tut.fi/~erkkila2/software/dsection/DSection.m</a></p>
  
    </div>

    <div class="Value">
      <h2>Value</h2>
      
      <p>A list with the following elements:</p>
  
      <p>MCData results from the MCMC estimation.
  x_LS Standard-least square estimate.
  groups factor defining the groups of samples, if
  any was provided. call the call to DSection.
  parameters a list of some of the parameters used
  in the estimation. p0 initial prior on
  proportions.</p>
  
    </div>

    <div class="Description">
      <h2>Description</h2>
      
      <p>The <em>DSection</em> algorithm performs gene expression
  deconvolution when priors on proportions are available,
  using a Markov Chain Monte Carlo approach <cite>Erkkila
  et al. (2010)</cite>.</p>
  
    </div>

    <div class="Details">
      <h2>Details</h2>
      
      <p>In <span class = "pkg">CellMix</span>, this method is registered with the key
  <code>'DSection'</code>, and is can be applied to gene
  expression data via the function <code><a href='ged.html'>ged</a></code>.</p>
  
      <p>This function uses the <span class = "pkg">RcppOctave</span> package to run
  the original Matlab code in <em>Octave</em>. The
  documentation was extracted from the Matlab source file,
  that can be found in the <span class = "pkg">CellMix</span> package
  "scripts/DSection" subdirectory.</p>
  
      <p>The Matlab code requires the Octave packages
  <em>statistics</em> and <em>optim</em> to run properly. These
  packages can be downloaded from Octave-forge:</p>
  
      <p><a href = 'http://sourceforge.net/projects/octave/files/Octave%20Forge%20Packages/Individual%20Package%20Releases/'>http://sourceforge.net/projects/octave/files/Octave Forge Packages/Individual Package Releases/</a></p>
  
      <p>and installed in Octave with:</p>
  
      <p><code>pkg install '<path/to/package/tar/gz/file>'</code></p>
  
      <p>or in R</p>
  
      <p><code>install.extras('CellMix', 'octave:DSection')</code></p>
  
    </div>

    <div class="References">
      <h2>References</h2>
      
      <p><a href = 'http://informatics.systemsbiology.net/DSection/'>http://informatics.systemsbiology.net/DSection/</a>
  <a href = 'http://www.cs.tut.fi/~erkkila2/software/dsection/index.html'>http://www.cs.tut.fi/~erkkila2/software/dsection/index.html</a></p>
  
      <p>Erkkila T, Lehmusvaara S, Ruusuvuori P, Visakorpi T,
  Shmulevich I and Lahdesmaki H (2010). "Probabilistic
  analysis of gene expression measurements from
  heterogeneous tissues." _Bioinformatics (Oxford,
  England)_, *26*(20), pp. 2571-7. ISSN 1367-4811, <URL:
  http://dx.doi.org/10.1093/bioinformatics/btq406>, <URL:
  http://www.ncbi.nlm.nih.gov/pubmed/20631160
  http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2951082\&tool=pmcentrez\&rendertype=abstract>.</p>
  
    </div>
    
    <h2 id="examples">Examples</h2>
    <pre class="examples"><div class='input'>#' # random global expression</div><div class='input'>x <- rmix(3, 50, 10)
</div><div class='input'>dim(x)
</div>
<div class='output'>  Features    Samples Components 
        50         10          3 
</div><div class='input'></div><div class='input'># extract true proportions</div><div class='input'>p <- coef(x)
</div><div class='input'># add noise to proportions</div><div class='input'>p0 <- scoef(abs(p + rmatrix(p, dist=rnorm, sd=0.15)))
</div><div class='input'># check how noisy this makes the proportion estimates</div><div class='input'>profplot(p, p0)
</div><p><img src='DSection-ex1.png' alt='' width='400' height='400' /></p><div class='input'></div><div class='input'>## <strong>Not run</strong>: </div><div class='input'># # Requires some extra Octave packages to be installed (it defines `gamfit`)</div><div class='input'># res <- DSection(exprs(x), p0, nSamples=100, nBurnIn=1L)</div><div class='input'># profplot(p, res$MCData$p)</div><div class='input'># ## <strong>End(Not run)</strong></div></pre>
  </div>
  <div class="span4">
    <!-- <ul>
      <li>DSection</li>
    </ul>
    <ul>
      
    </ul> -->
      
    
    <h2>Author</h2>
    
  Original Matlab code: Timo Erkkila

  Wrapper function: Renaud Gaujoux

    
  </div>
</div>
      
      <footer>
      <p class="pull-right"><a href="#">Back to top</a></p>
<p>Built by <a href="https://github.com/hadley/staticdocs">staticdocs</a>. Styled with <a href="http://twitter.github.com/bootstrap">bootstrap</a>.</p>
      </footer>
    </div>
  </body>
</html>