<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
<title>gedAlgorithm.nnls. CellMix 1.5.3</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="author" content="">

<!-- bootstrap css -->
<link href="css/bootstrap.css" rel="stylesheet">
<link href="css/bootstrap-responsive.css" rel="stylesheet">
<!-- highlighting css -->
<link href="css/highlight.css" rel="stylesheet">
<!-- custom css -->
<link href="css/staticdocs.css" rel="stylesheet">

<!-- bootstrap javascript -->
<script src="js/jquery.min.js"></script>
<script src="js/bootstrap.min.js"></script>
<!-- d3 word cloud -->
<script src="js/d3.v2.min.js"></script>
<script src="js/d3.layout.cloud.js"></script>

<!--[if lt IE 9]>
  <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
<![endif]-->
  </head>

  <body>
    <div class="navbar">
  <div class="navbar-inner">
    <div class = "container">
      <a class="brand" href="index.html">CellMix 1.5.3</a>
      <div class="nav">
        <ul class="nav">
        <li><a href="_MAN.html"><i class="icon-home icon-white"></i> Index</a></li>
       </ul>
   	  </div>
    </div>
  </div>
</div>

    <div class="container">
      <header>
        
      </header>
      
      <h1>Partial Gene Expression Deconvolution by Nonnegative Least-Squares</h1>

<div class="row">
  <div class="span8">
    <h2>Usage</h2>
    <pre>gedAlgorithm.nnls(..., rescale = TRUE)</pre>
    
    <h2>Arguments</h2>
    <dl>
      <dt>...</dt>
      <dd>extra arguments passed to
  <code><a href='http://www.inside-r.org/r-doc/NMF/dot-fcnnls'>.fcnnls</a></code>.</dd>
      <dt>rescale</dt>
      <dd>logical used when estimating proportions
  from signatures, that indicates if the esti,ated
  coefficients should be scaled to sum up to one
  (<code>TRUE</code>) or left as estimated by the linear
  regression (<code>FALSE</code>). This scaling is performed
  after the coefficients have been forced to be
  nonnegative.</dd>
    </dl>
    
    <div class="Description">
      <h2>Description</h2>
      
      <p>Estimates cell/tissue proportions given a known set of
  cell/tissue-specific expression signatures, using the
  fast combinatorial nonnegative least-square method, which
  was proposed by <cite>Kim et al. (2007)</cite> to perform
  nonnegative matrix factorization of gene expression --
  not originally for deconvolution.</p>
  
    </div>

    <div class="Details">
      <h2>Details</h2>
      
      <p>All regressions are fitted using the function
  <code><a href='http://www.inside-r.org/r-doc/NMF/dot-fcnnls'>.fcnnls</a></code>.</p>
  
    </div>

    <div class="References">
      <h2>References</h2>
      
      <p>Kim H and Park H (2007). "Sparse non-negative matrix
  factorizations via alternating non-negativity-constrained
  least squares for microarray data analysis."
  _Bioinformatics (Oxford, England)_, *23*(12), pp.
  1495-502. ISSN 1460-2059, <URL:
  http://dx.doi.org/10.1093/bioinformatics/btm134>, <URL:
  http://www.ncbi.nlm.nih.gov/pubmed/17483501>.</p>
  
    </div>
    
    <h2 id="examples">Examples</h2>
    <pre class="examples"><div class='input'># random target matrix</div><div class='input'>x <- rmatrix(100, 20)
</div><div class='input'># random cell signatures</div><div class='input'>s <- rmatrix(100, 3)
</div><div class='input'></div><div class='input'># deconvolve using nonnegative least-squares</div><div class='input'>res <- ged(x, s, 'nnls')
</div><div class='input'>coef(res)
</div>
<div class='output'>          [,1]     [,2]      [,3]      [,4]      [,5]      [,6]      [,7]      [,8]      [,9]     [,10]     [,11]     [,12]     [,13]
[1,] 0.4488777 0.333963 0.4528069 0.3270806 0.5349628 0.4684036 0.3864849 0.4368379 0.3859660 0.4208052 0.4233316 0.2523020 0.3470275
[2,] 0.1850926 0.238325 0.3607510 0.1867547 0.2678708 0.2546234 0.2789545 0.1698202 0.2947573 0.3971309 0.2457456 0.4143946 0.3521411
[3,] 0.3660297 0.427712 0.1864421 0.4861648 0.1971664 0.2769730 0.3345606 0.3933418 0.3192767 0.1820639 0.3309229 0.3333034 0.3008313
         [,14]     [,15]     [,16]     [,17]     [,18]     [,19]     [,20]
[1,] 0.2105858 0.1359857 0.5231282 0.4191761 0.3196914 0.4833751 0.4079663
[2,] 0.2684163 0.3481854 0.1950748 0.1868577 0.2782089 0.3405490 0.1347825
[3,] 0.5209979 0.5158289 0.2817970 0.3939662 0.4020997 0.1760758 0.4572512
</div><div class='input'># signatures are not updated</div><div class='input'>identical(basis(res), s)
</div>
<div class='output'>[1] TRUE
</div><div class='input'></div></pre>
  </div>
  <div class="span4">
    <!-- <ul>
      <li>gedAlgorithm.nnls</li><li>nnls-ged</li>
    </ul>
    <ul>
      
    </ul> -->
      
        
  </div>
</div>
      
      <footer>
      <p class="pull-right"><a href="#">Back to top</a></p>
<p>Built by <a href="https://github.com/hadley/staticdocs">staticdocs</a>. Styled with <a href="http://twitter.github.com/bootstrap">bootstrap</a>.</p>
      </footer>
    </div>
  </body>
</html>