<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
<title>gedAlgorithm.DSection. CellMix 1.5.1</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="author" content="">

<!-- bootstrap css -->
<link href="css/bootstrap.css" rel="stylesheet">
<link href="css/bootstrap-responsive.css" rel="stylesheet">
<!-- highlighting css -->
<link href="css/highlight.css" rel="stylesheet">
<!-- custom css -->
<link href="css/staticdocs.css" rel="stylesheet">

<!-- bootstrap javascript -->
<script src="js/jquery.min.js"></script>
<script src="js/bootstrap.min.js"></script>
<!-- d3 word cloud -->
<script src="js/d3.v2.min.js"></script>
<script src="js/d3.layout.cloud.js"></script>

<!--[if lt IE 9]>
  <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
<![endif]-->
  </head>

  <body>
    <div class="navbar">
  <div class="navbar-inner">
    <div class = "container">
      <a class="brand" href="index.html">CellMix 1.5.1</a>
      <div class="nav">
        <ul class="nav">
        <li><a href="_MAN.html"><i class="icon-home icon-white"></i> Index</a></li>
       </ul>
   	  </div>
    </div>
  </div>
</div>

    <div class="container">
      <header>
        
      </header>
      
      <h1>Partial Gene Expression Deconvolution with DSection</h1>

<div class="row">
  <div class="span8">
    <h2>Usage</h2>
    <pre>gedAlgorithm.DSection(..., maxIter = 500L, p0, data = NULL, nBurnIn = 4 * maxIter, 
  W0 = max(100, nbasis(seed)), W_proposal = 100, samplep = TRUE)

<span class="label label-success">S3</span> (ged_DSection)
`csTopTable`(x, coef = 1L, decreasing = TRUE, ...)</pre>
    
    <h2>Arguments</h2>
    <dl>
      <dt>data</dt>
      <dd>variable (e.g., factor) that defines groups
  of samples. Cell-specific signatures will be computed for
  each group.</dd>
      <dt>maxIter</dt>
      <dd>number of sampling, not including burn-in
  sampling.</dd>
      <dt>...</dt>
      <dd>extra arguments passed to
  <code><a href='DSection.html'>DSection</a></code>.</dd>
      <dt>coef</dt>
      <dd>specifies the reference cell type or group of
  samples, for which differential expression is computed.</dd>
      <dt>p0</dt>
      <dd><T-by-J> matrix of prior predictions on cell
  type proportions. T is the number of cell types, and
  columns in <code>p0</code> must be positive and add up to one.</dd>
      <dt>nBurnIn</dt>
      <dd>Amount of burn-in. NOTE: keep nBurnIn >
  0.</dd>
      <dt>W0</dt>
      <dd>Prior prediction weight, i.e., degree of
  confidence, on <code>p0</code>. Defines the peakedness of
  Dirichlet density around p0. NOTE: keep W0 >= T.</dd>
      <dt>W_proposal</dt>
      <dd>Transition kernel weight, defines the
  peakedness of Dirichlet density around p*, the old value.
  The higher W_proposal is, the smaller the proposal steps
  around p* are.</dd>
      <dt>samplep</dt>
      <dd>logical value, indicating whether to
  sample from the posterior for cell type proportions
  (<code>TRUE</code>) or not (<code>FALSE</code>). SUGGESTED USE:
  sample from the posterior (samplep = 1).</dd>
      <dt>x</dt>
      <dd>data object, typically returned by
  <code><a href='ged.html'>ged</a></code>.</dd>
      <dt>decreasing</dt>
      <dd>logical that indicates the feature
  ordering, based on their p-values or FDRs.</dd>
    </dl>
    
    <div class="Description">
      <h2>Description</h2>
      
      <p>Estimates cell/tissue cell/tissue-specific expression
  signatures, given proportion priors using the MCMC
  approach from <cite>Erkkila et al. (2010)</cite>, implemented
  in Matlab and wrapped in <span class = "pkg">CellMix</span> by the function
  <code><a href='DSection.html'>DSection</a></code>.</p>
  
      <p>The S3 method <code>csTopTable</code> for DSection fits
  computes nominal p-values (i.e. unadjusted) of
  differential expression between cell type or group of
  samples within each cell type, for deconvolution results
  from the <code><a href='gedAlgorithm.DSection.html'>DSection</a></code> algorithm.</p>
  
    </div>

    <div class="References">
      <h2>References</h2>
      
      <p>Erkkila T, Lehmusvaara S, Ruusuvuori P, Visakorpi T,
  Shmulevich I and Lahdesmaki H (2010). "Probabilistic
  analysis of gene expression measurements from
  heterogeneous tissues." _Bioinformatics (Oxford,
  England)_, *26*(20), pp. 2571-7. ISSN 1367-4811, <URL:
  http://dx.doi.org/10.1093/bioinformatics/btq406>, <URL:
  http://www.ncbi.nlm.nih.gov/pubmed/20631160
  http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2951082\&tool=pmcentrez\&rendertype=abstract>.</p>
  
    </div>
    
    <h2 id="examples">Examples</h2>
    <pre class="examples"><div class='input'># random global expression</div><div class='input'>x <- rmix(3, 50, 20)
</div><div class='input'># extract true proportions</div><div class='input'>p <- coef(x)
</div><div class='input'># add noise</div><div class='input'>p0 <- scoef(abs(p + rmatrix(p, dist=rnorm, sd=0.15)))
</div><div class='input'># check how noisy this makes the proportion estimates</div><div class='input'>profplot(p, p0)
</div><p><img src='gedAlgorithm.DSection-ex1.png' alt='' width='400' height='400' /></p><div class='input'></div><div class='input'># deconvolve using DSection</div><div class='input'>## <strong>Not run</strong>: </div><div class='input'># res <- ged(x, p0, 'DSection', maxIter=10, seed=12345)</div><div class='input'># head(basis(res))</div><div class='input'># # proportions are updated</div><div class='input'># !identical(coef(res), p0)</div><div class='input'># # check how better they are</div><div class='input'># profplot(x, res)</div><div class='input'># </div><div class='input'># \dontshow{</div><div class='input'># 	stopifnot( !identical(coef(res), p0) )</div><div class='input'># 	stopifnot( nmf.equal(res, ged(x, p0, 'DSection', maxIter=10, seed=12345)) )</div><div class='input'># }</div><div class='input'># ## <strong>End(Not run)</strong></div></pre>
  </div>
  <div class="span4">
    <!-- <ul>
      <li>csTopTable.ged_DSection</li><li>DSection-ged</li><li>gedAlgorithm.DSection</li>
    </ul>
    <ul>
      
    </ul> -->
      
        
  </div>
</div>
      
      <footer>
      <p class="pull-right"><a href="#">Back to top</a></p>
<p>Built by <a href="https://github.com/hadley/staticdocs">staticdocs</a>. Styled with <a href="http://twitter.github.com/bootstrap">bootstrap</a>.</p>
      </footer>
    </div>
  </body>
</html>