<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
<title>csSAM-package. csSAM 1.2.4</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="author" content="
  Shai Shen-Orr, Rob Tibshirani, Narasimhan
  Balasubramanian, David Wang

  Maintainer: Shai Shen-Orr &lt;shenorr@stanford.edu&gt;
">

<!-- bootstrap css -->
<link href="css/bootstrap.css" rel="stylesheet">
<link href="css/bootstrap-responsive.css" rel="stylesheet">
<!-- highlighting css -->
<link href="css/highlight.css" rel="stylesheet">
<!-- custom css -->
<link href="css/staticdocs.css" rel="stylesheet">

<!-- bootstrap javascript -->
<script src="js/jquery.min.js"></script>
<script src="js/bootstrap.min.js"></script>
<!-- d3 word cloud -->
<script src="js/d3.v2.min.js"></script>
<script src="js/d3.layout.cloud.js"></script>

<!--[if lt IE 9]>
  <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
<![endif]-->
  </head>

  <body>
    <div class="navbar">
  <div class="navbar-inner">
    <div class = "container">
      <a class="brand" href="index.html">csSAM 1.2.4</a>
      <div class="nav">
        <ul class="nav">
        <li><a href="_MAN.html"><i class="icon-home icon-white"></i> Index</a></li>
       </ul>
   	  </div>
    </div>
  </div>
</div>

    <div class="container">
      <header>
        
      </header>
      
      <h1>Cell-specific Differential Expression (csSAM)</h1>

<div class="row">
  <div class="span8">
        
    <div class="Description">
      <h2>Description</h2>
      
      <p>SAM for Cell-specific Differential Expression SAM.</p>
  
    </div>

    <div class="Details">
      <h2>Details</h2>
      
      <p><table><tr><td> Package: </td>
<td> csSAM</td>
</tr><tr><td> Type: </td>
<td>
  Package</td>
</tr><tr><td> Version: </td>
<td> 1.2</td>
</tr><tr><td> Date: </td>
<td> 2011-10-08</td>
</tr><tr><td>
  License: </td>
<td> LGPL</td>
</tr><tr><td> LazyLoad: </td>
<td> yes</td>
</tr><tr><td></p>
  
      <p>Tissues are often made up of multiple cell-types. Each
  with its own functional attributes and molecular
  signature.  Yet, the proportions of any given cell-type
  in a sample can vary markedly. This results in a
  significant loss of sensitivity in gene expression
  studies and great difficulty in identifying the cellular
  source of any perturbations. Here we present a
  statistical methodology (cell-type specific Significance
  Analysis of Microarrays or csSAM) which, given microarray
  data from two groups of biological samples and the
  relative cell-type frequencies of each sample, estimates
  in a virtual manner the gene expression data for each
  cell-type at a group level, and uses these to identify
  differentially expressed genes at a cell-type specific
  level between groups.</p>
  
      <p>The lower limit for the number of samples needed for
  deconvolving the cell-specific expression of N cell-types
  is N+1. For a singe color array - the result could be
  interperted as the avg. expression level of a given gene
  in a cell-type of that group. Multiplied by the frequecy
  of a given cell-type in an individual in the group, it is
  the amount contributed by that cell type to the overall
  measured expression on the array.  </td>
<td> Key functions for
  this package:</td>
</tr><tr><td> csSamWrapper - Single wrapper function
  performs all functionality. csfit: For deconvolving the
  average cell-type specific expression for each cell-type
  in a given group.</td>
<td> csSAM: For calculating the constrast
  between every pair of cells being compared between the
  two groups.</td>
</tr><tr><td> fdrCsSAM: Estimate the false discovery
  rate for each cell-type specific comparison.</td>
<td>
  findSigGenes:Identifies the list of differentially
  expressed genes in a given cell-type at a given FDR
  cutoff.</td>
</tr><tr><td> plotCsSAM:Plots a fdr plot of ther results.</td>
<td> Package: </td>
</tr></table> Additional functions exists (runSAM and fdrSAM to
  contrast csSAM with the tissue heterogeneity ignorant
  SAM).</p>
  
    </div>

    <div class="References">
      <h2>References</h2>
      
      <p>Shen-Orr SS, Tibshirani R, Khatri P, Bodian DL, Staedtler
  F, Perry NM, Hastie T, Sarwal MM, Davis MM and Butte AJ
  (2010). "Cell type-specific gene expression differences
  in complex tissues." _Nature methods_, *7*(4), pp. 287-9.
  ISSN 1548-7105, <URL:
  http://dx.doi.org/10.1038/nmeth.1439>, <URL:
  http://www.ncbi.nlm.nih.gov/pubmed/20208531>.</p>
  
    </div>
    
    <h2 id="examples">Examples</h2>
    <pre class="examples"><div class='input'>library("csSAM")
</div><div class='input'>##</div><div class='input'>## Generate random dataset</div><div class='input'>##</div><div class='input'>set.seed(143)
</div><div class='input'>k <- 5 # number of cell types
</div><div class='input'>ng <- 500 # number of genes
</div><div class='input'>p <- 20 # number of samples
</div><div class='input'>ndiff <- 100 # number of genes differentially expressed
</div><div class='input'></div><div class='input'># true cell-specific signatures</div><div class='input'>H1 <- matrix(rnorm(5*ng), ncol=ng)
</div><div class='input'>H2 <- H1
</div><div class='input'># create differential expression for 3rd cell type</div><div class='input'>H2[3,1:ndiff] <- H2[3,1:ndiff] + 5
</div><div class='input'></div><div class='input'># cell frequency matrix per sample</div><div class='input'>cc <- matrix(runif(p*k), ncol=k)
</div><div class='input'>cc <- t(scale(t(cc), center=FALSE, scale=rowSums(cc)))
</div><div class='input'>colnames(cc) <- paste('cellType', 1:ncol(cc), sep="")
</div><div class='input'></div><div class='input'># global expression matrix</div><div class='input'>G <- rbind(cc[1:10, ] %*% H1, cc[11:p, ] %*%H2 ) + matrix(rnorm(p*ng), ncol=ng)
</div><div class='input'># sample classes (2 groups)</div><div class='input'>y <- gl(2, p/2)
</div><div class='input'></div><div class='input'>fileName = "Example File.pdf";
</div><div class='input'></div><div class='input'></div><div class='input'># Now run, either using the wrapper</div><div class='input'># NB: more permutations would be needed for real data</div><div class='input'>deconvResults = csSamWrapper(G, cc, y, nperms = 50, alternative = "two.sided", standardize = TRUE, medianCenter = TRUE,fileName = fileName)
</div>
<strong class='warning'>Warning message:
1 x value <= 0 omitted from logarithmic plot</strong>
<strong class='warning'>Warning message:
1 x value <= 0 omitted from logarithmic plot</strong>
<strong class='warning'>Warning message:
1 x value <= 0 omitted from logarithmic plot</strong>
<strong class='warning'>Warning message:
1 x value <= 0 omitted from logarithmic plot</strong>
<strong class='warning'>Warning message:
1 x value <= 0 omitted from logarithmic plot</strong>
<strong class='warning'>Warning message:
1 x value <= 0 omitted from logarithmic plot</strong><div class='input'></div><div class='input'># Or by calling each function independently</div><div class='input'># (e.g. useful if you want to perform only cell-specific expression without differential expression).</div><div class='input'>## <strong>Not run</strong>: </div><div class='input'># numset = nlevels(y)</div><div class='input'># n <- summary(y, maxsum=Inf) # number of samples in each class</div><div class='input'># numgene = ncol(G)</div><div class='input'># numcell = ncol(cc)</div><div class='input'># geneID = colnames(G)</div><div class='input'># cellID = colnames(cc)</div><div class='input'># </div><div class='input'># deconv <- list()</div><div class='input'># # run analysis</div><div class='input'># for (curset in levels(y))</div><div class='input'># 	deconv[[curset]]= csfit(cc[y==curset,], G[y==curset,])</div><div class='input'># </div><div class='input'># rhat <- array(dim = c(numcell,numgene))</div><div class='input'># rhat[, ] <- csSAM(deconv[[1]]$ghat, deconv[[1]]$se,</div><div class='input'>#                   n[1], deconv[[2]]$ghat, deconv[[2]]$se, n[2],</div><div class='input'>#                   standardize=TRUE, medianCenter=TRUE, nonNeg=TRUE)</div><div class='input'># </div><div class='input'># tt.sam <- runSAM(G, y)</div><div class='input'># falseDiscovR <- fdrCsSAM(G,cc,y,n,numcell,numgene, rhat,</div><div class='input'>#                     nperms = 200,standardize=TRUE,alternative='two.sided',</div><div class='input'>#                     medianCenter=TRUE, nonNeg=TRUE)</div><div class='input'># falseDiscovRSAM <- fdrSAM(G, y, nperms=200, alternative = 'two.sided',tt.sam)</div><div class='input'># sigGene <- findSigGene(G, cc, y, rhat, falseDiscovR)</div><div class='input'># </div><div class='input'># plotCsSAM(falseDiscovR, falseDiscovRSAM,alternative='two.sided',cellID,numcell, fileName)</div><div class='input'># print (falseDiscovR$fdr.g[ , ] )</div><div class='input'># ## <strong>End(Not run)</strong></div></pre>
  </div>
  <div class="span4">
    <!-- <ul>
      <li>csSAM-package</li>
    </ul>
    <ul>
      
    </ul> -->
      
    
    <h2>Author</h2>
    
  Shai Shen-Orr, Rob Tibshirani, Narasimhan
  Balasubramanian, David Wang

  Maintainer: Shai Shen-Orr <shenorr@stanford.edu>

    
  </div>
</div>
      
      <footer>
      <p class="pull-right"><a href="#">Back to top</a></p>
<p>Built by <a href="https://github.com/hadley/staticdocs">staticdocs</a>. Styled with <a href="http://twitter.github.com/bootstrap">bootstrap</a>.</p>
      </footer>
    </div>
  </body>
</html>